docker/package/tezos_setup_wizard.py

```diff
@@ -232,6 +232,35 @@ def check_blockchain_data(self):
             return False
         return True
 
+    # Check <network>.xtz-shots.io/base.json and collect the most recent snapshot
+    # that is suited for the chosen history mode
+    def get_snapshot_link(self):
+        self.config["snapshot_url"] = None
+        json_url = "https://xtz-shots.io/base.json"
+        try:
+            with urllib.request.urlopen(json_url) as url:
+                snapshot_array = json.load(url)
+                snapshot_array.reverse()
```

Comment:
IIUC this is done this so that the most recent snapshot will be on top, right?

If so, wouldn't it be better to sort them by `block_height` instead, which would be a more stable approach?